### PR TITLE
[improve][doc] add workflow for previewing content

### PR DESCRIPTION
**Pull request by Anonymitaet**
The preview looks good:

image

image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
cc @momo-jun @DaveDuggins @D-2-Ed

File: site2/website-next/contribute/document-preview.md [hunk 1]
```diff
@@ -37,14 +37,22 @@ Follow these steps to preview documentation changes on the **master** branch.
...
-    sh start.sh [<version-number> ...]
+
+    # Preview changes on a specific version version
+    sh start.sh 2.10.x 
+
+    # Preview changes on multiple versions
+    sh start.sh 2.10.x 2.9.x ...
     ```
 
 3. By default, a browser window will open at http://localhost:3000 to show the changes.
```
Comment (momo-jun) [inline site2/website-next/contribute/document-preview.md:50]: Suggest removing the numbering `3` here. It's the result of step 2, not user's behavior.

**Comment by Anonymitaet** [comment]
@tisonkun @momo-jun I've updated based on your comments, PTAL, thank you!

File: site2/website-next/contribute/document-preview.md [hunk 2]
```diff
@@ -37,13 +37,21 @@ Follow these steps to preview documentation changes on the **master** branch.
 2. Run the following command to preview changes:
 
     ```bash
-    # Preview the next version
+    # Preview changes on latest stable version
```
Comment (tisonkun) [inline site2/website-next/contribute/document-preview.md:40]:
```suggestion
    # Preview changes on latest stable version
```

File: site2/website-next/contribute/document-preview.md [hunk 3]
```diff
@@ -37,13 +37,21 @@ Follow these steps to preview documentation changes on the **master** branch.
...
+    # Preview changes on multiple versions
+    sh start.sh 2.10.x 2.9.x ...
     ```
+    
+  By default, a browser window will open at http://localhost:3000 to show the changes.
 
-3. By default, a browser window will open at http://localhost:3000 to show the changes.
+  > **Tip**
+  >
+  > If you want to preview changes on `master`, change the URL to `http://localhost:3000/docs/next`.
```
Comment (tisonkun) [inline site2/website-next/contribute/document-preview.md:54]:
```suggestion
:::tip

If you want to preview changes on `master`, change the URL to `http://localhost:3000/docs/next`.

:::
```

Let's use admonitions consistently. Also IIRC the admonitions rendering will fail if leading idents exist.

Comment (tisonkun) [reply site2/website-next/contribute/document-preview.md:54]: The original content seems also rendered wrongly:

image

Please preview locally :)

Comment (tisonkun) [reply site2/website-next/contribute/document-preview.md:54]: Emmm...It seems a two-space indent can be rendered properly:

image

But we may unindent for both this tip and the following screenshot?

**Comment by Anonymitaet** [comment]
@tisonkun 

>Let's use admonitions consistently.

Nice catch! Done.

>But we may unindent for both this tip and the following screenshot?

I indent them with 2 spaces as they belong to the step 2.
image

This is the corresponding preview:
image

**Comment by tisonkun** [comment]
@Anonymitaet It seems that you don't push your local commits to the remote branch :)

**Comment by Anonymitaet** [comment]
@tisonkun Ah! Sorry, my bad. Already pushed.

File: site2/website-next/contribute/document-preview.md [hunk 4]
```diff
@@ -37,15 +37,25 @@ Follow these steps to preview documentation changes on the **master** branch.
 2. Run the following command to preview changes:
 
     ```bash
-    # Preview the next version
+    # Preview changes on latest stable version
```
Comment (tisonkun) [inline site2/website-next/contribute/document-preview.md:40]:
```suggestion
    # Preview changes on latest stable version
```

**Review by tisonkun — APPROVED** [review]
LGTM. Merging...

**Pull request merged**